pepper-apis/src/main/java/org/broadinstitute/ddp/cache/LanguageStore.java

```diff
@@ -14,31 +17,26 @@
 
     public static final String DEFAULT_LANG_CODE = "en";
 
-    private static final Map<String, LanguageDto> languages = new HashMap<>();
-
-    public static LanguageDto get(String isoCode) {
-        return languages.get(isoCode);
-    }
+    private static Map<String, LanguageDto> languages = Collections.emptyMap();
 
-    public static LanguageDto getDefault() {
-        return languages.get(DEFAULT_LANG_CODE);
+    public static synchronized void init(Handle handle) {
+        Map<String, LanguageDto> map = new HashMap<>();
+        handle.attach(JdbiLanguageCode.class)
+                .findAll()
+                .forEach(lang -> map.put(lang.getIsoCode(), lang));
+        languages = Map.copyOf(map);
     }
 
-    public static LanguageDto getOrCompute(Handle handle, String isoCode) {
-        return languages.computeIfAbsent(isoCode, key ->
-                handle.attach(JdbiLanguageCode.class).findLanguageDtoByCode(isoCode));
+    public static LanguageDto get(@Nullable String isoCode) {
```

Comment:
Have we run into cases where `isoCode` is passed a `null` value?

Reply:
Yes, I ran into issue where some of the Profile route code is calling it with `null` and apparently the underlying immutable map method will barf on a null...